### PR TITLE
Update issues.rst

### PR DESCRIPTION
**Pull request by alat-rights**
# issues.rst Pull Request

The table was borked. This should fix it.

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

### CODE REVIEW

File: docs/source/get_started/issues.rst [hunk 1]
```diff
@@ -13,22 +13,22 @@ broken.
 *Note: This list is likely to be non-exhaustive. If we missed something, 
 please let us know [here](https://github.com/deepchem/deepchem/issues/2376).*
 
-+--------------------------------+-------------------+---------------------------------------------------+
-| Feature                        | Deepchem response | Tracker and notes                                 |
-|                                |                   |                                                   |
-+================================+===================+===================================================+
-| AtomicConvModel                | Severe            | [pull#2374](https://github.com/deepchem/deepchem/pull/2374)                   |
```
Comment (rbharath) [inline docs/source/get_started/issues.rst:20]: We can remove both `AtomicConvModel` and `RdkitGridFeaturizer` thanks to @ncfrey's hard work!

**Review by rbharath — COMMENTED** [review]
We should update since deepchem 2.5 has fixed some of these issues :)

**Comment by alat-rights** [comment]
Got it. I'll try to track down the release notes.

Should I put updating this page in release checklist?

**Comment by rbharath** [comment]
The release notes are at https://forum.deepchem.io/t/deepchem-2-5-0-release/439 (but don't go into full depth on the fixes here). I think adding updating this page to the release checklist would be a great idea!

File: docs/source/get_started/issues.rst (hunk 1 above)
Comment (alat-rights) [reply docs/source/get_started/issues.rst:20]: Oh my gosh AtomicConv is fixed? That's amazing! There's a couple of issues and PRs we could close then

**Comment by alat-rights** [comment]
That should fix it? :)

**Review by rbharath — APPROVED** [review]
Feel free to merge once tests finish running :)

**Pull request merged**